.github/workflows/tests_secondary.yml

```diff
@@ -110,17 +110,27 @@ jobs:
         path: test-results
 
   test-package-installations:
-    runs-on: ubuntu-20.04
+    name: "Installation Test ${{ matrix.os }} (${{ matrix.node_version }})"
+    runs-on: ${{ matrix.os  }}
     strategy:
       fail-fast: false
       matrix:
+        os:
+        - ubuntu-latest
+        - macos-latest
+        - windows-latest
         node_version:
-        - "12.0.0"
-        - "^12.0.0"
-        - "^14.1.0"  # pre 14.1, zip extraction was broken (https://github.com/microsoft/playwright/issues/1988)
-        - "^16.0.0"
         - "^18.0.0"
-    timeout-minutes: 20
+        include:
+        - os: ubuntu-latest
+          node_version: "12.0.0"
```

Comment:
make sure to rebase correctly. Seems like you have to delete 12.x and can add 18.x back.